### ISSUE TITLE
Add tests for check_incmd

### ISSUE THREAD

**Issue by philsmart** [ukf/ukf-testbed#60]

**Comment by philsmart** [comment]
I am unable to trigger:
```
    <xsl:template match="@incmd:contactType[not(starts-with(.,'http://'))][not(starts-with(.,'https://'))]">
        <xsl:call-template name="error">
            <xsl:with-param name="m">incmd:contactType must be an absolute URI</xsl:with-param>
        </xsl:call-template>
    </xsl:template>
```

If the rule that checks for specific values exists:
```
    <!--
        Check for specific values.  This test is probably over-specific in the long term.
    -->
    <xsl:template match="@incmd:contactType
        [not(. = 'http://id.incommon.org/metadata/contactType/security')]">
        <xsl:call-template name="error">
            <xsl:with-param name="m">
                <xsl:text>unknown value '</xsl:text>
                <xsl:value-of select="."/>
                <xsl:text>' for incmd:contactType</xsl:text>
            </xsl:with-param>
        </xsl:call-template>
    </xsl:template>
```
This seems to work correctly if you switch the ordering of them around, so the specific values check is before the absolute URI check.

**Comment by iay** [comment]
The fact that moving them around helped indicates that this might be a rule priority issue, in which case moving them around would be the wrong answer. You'd be hiding the bug, not fixing it.

If that comment doesn't make sense, schedule ten minutes and we can talk about this horrible, horrible aspect of XSLT.

**Comment by philsmart** [comment]
I'd appreciate the chat about it. I'll ping you tomorrow to schedule something.

**Comment by philsmart** [comment]
I attempted to fix the rule priority issue by 'ordering' the templates in the file.  I did not know that they were triggering an error (more than one matching template after [conflict resolution](https://www.w3.org/TR/xslt-10/#conflict)) and that may 'fix' relied on an implementation detail of the XSLT processor used, i.e. 'An XSLT processor may signal the error; if it does not signal the error, it must recover by choosing, from amongst the matching template rules that are left, the one that occurs last in the stylesheet'.
